packages/std/src/errors/std_error.rs

```diff
@@ -765,8 +765,10 @@ mod tests {
 
     #[test]
     fn from_std_str_utf8error_works() {
-        let broken = Vec::from(b"Hello \xF0\x90\x80World" as &[u8]);
-        let error: StdError = str::from_utf8(&broken).unwrap_err().into();
+        let broken = b"Hello \xF0\x90\x80World";
+        #[allow(unknown_lints)]
+        #[allow(invalid_from_utf8)]
+        let error: StdError = str::from_utf8(broken).unwrap_err().into();
```

Comment:
I think this diff id not needed anymore. The version on latest main should make clippy 1.75.0 happy.